src/common/Surface.js

```diff
@@ -130,13 +138,43 @@
             .display(this.showTitle())
             .render()
         ;
+
+        var surfaceButtons = element.selectAll(".surface-button").data(this.buttonAnnotations());
+        surfaceButtons.enter().append("tspan")
```

Comment:
When you append the "tspan" you should set its class to "surface-button" (as this is what you are using for the selector - otherwise subsequent calls won't align the data correctly.